traffic_ops/experimental/go-api/api.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+/*
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+/*
+This module it provides a server creating a RESTful API towards Traffic-Ops DB.
+It was created in order to POC the concept of API Gateway (added on another PR) between the different modules of Traffic-Ops, specifically old perl Traffic-Ops, and new (go?) one.
+
+At first step, only "tenant" table is covered (CRUD).
+Other tables can be added, by adding modules (see tenant foe example), and seeding their enpoints below.
+To Run this module, call: go run api.go --server --db-config-file 
+e.g. go run api.go --server :8888 --db-config-file ../../app/conf/test/database.conf
```

Comment:
Consider printing this text as an example, when invalid or empty arguments are passed to the application? That way, users can see how to run the app, without having to look at the source code.